WhatsApp_DP_download/readme.md

```diff
@@ -0,0 +1,8 @@
+<p align="center"><img src="https://github.com/sudipg4112001/Amazing-Python-Scripts/blob/DP_download/WhatsApp_DP_download/Image.jpg"/></p>
+
+### About this feature:
+
+This is purely a python script to the get profile photos rather download profile photos of all the contacts in our whatsapp contact list.
+
+
+### This, is a gist of this project..!!
```

Comment:
I said to remove this line: "### This, is a gist of this project..!!" and not the above text. Bring back that text and remove this